src/tracing/internal/track_event_internal.cc

```diff
@@ -312,19 +315,18 @@ bool TrackEventInternal::IsCategoryEnabled(
   };
 
   // First try exact matches, then pattern matches.
-  const std::array<MatchType, 2> match_types = {
-      {MatchType::kExact, MatchType::kPattern}};
+  const std::array<MatchType, 3> match_types = {
+      {MatchType::kExact, MatchType::kPattern, MatchType::kWildcard}};
   for (auto match_type : match_types) {
-    // 1. Enabled categories.
-    if (NameMatchesPatternList(config.enabled_categories(), category.name,
+    // 1. Disabled categories.
```

Comment:
Wondering if disabled > enabled is what we want.. :)

`disabled_categories: "some_categories*", enabled_categories: "some_categories_i_want_*"`
Would mean `some_categories_i_want_*` categories stay disabled.

I think it'd also mean a trace event with two categories "navigation" & "rendering" would stay disabled with `disabled_categories: "navigation", enabled_categories: "rendering"`. Maybe that's what we would want, because why specify "navigation" explicitly as disabled?

@ddiproietto wdyt?